tests/e2e/helpers/dumpinfo.go

```diff
@@ -58,11 +59,15 @@ func DumpInfo(ctx context.Context, cfg *envconf.Config) (context.Context, error)
 	}
 	r := client.Resources(opts.Namespace)
 
+	if err := dumpPodSummary("pods.txt", exportDir); err != nil {
+		klog.ErrorS(err, "Failed to dump pod summary")
+	}
+
 	podList := &corev1.PodList{}
 	if err = r.List(
 		ctx,
 		podList,
-		resources.WithLabelSelector("app.kubernetes.io/instance="+opts.DaemonSetName),
+		resources.WithLabelSelector("app.kubernetes.io/name="+opts.DaemonSetName),
```

Comment:
unrelated to this change, but should we change app.kubernetes.io/instance -> app.kubernetes.io/part-of 

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
ah ignore me, just checked out the current helm chart, part-of label is used in object labels, instance label is for selector.